### PR TITLE
Beautified Welcome Screen

### PR DESCRIPTION
**Pull request by R-Ramana**
Made the following changes

- Logo for NotUS
- Line breaks between commands to make it easier for the user

### CODE REVIEW

File: src/main/java/seedu/duke/command/ListNoteCommand.java [hunk 1]
```diff
@@ -192,7 +192,7 @@ public String execute() {
         } else {
             noteString = getSortedString(sortedTaggedNotes);
         }
-        return COMMAND_SUCCESSFUL_MESSAGE + noteString.toString();
+        return InterfaceManager.LS + COMMAND_SUCCESSFUL_MESSAGE + noteString.toString();
```
Comment (Chongjx) [inline src/main/java/seedu/duke/command/ListNoteCommand.java:195]:
```suggestion
        return Formatter.LS + COMMAND_SUCCESSFUL_MESSAGE + noteString.toString();
```

File: src/main/java/seedu/duke/command/ListNoteCommand.java [hunk 2]
```diff
@@ -156,7 +156,7 @@ public String execute() {
 
         // If the user inputted tags does not match any of the existing tags.
         if (tagList.isEmpty()) {
-            return COMMAND_UNSUCCESSFUL_MESSAGE_INVALID_TAG;
+            return InterfaceManager.LS + COMMAND_UNSUCCESSFUL_MESSAGE_INVALID_TAG;
```
Comment (Chongjx) [inline src/main/java/seedu/duke/command/ListNoteCommand.java:159]:
```suggestion
            return Formatter.LS + COMMAND_UNSUCCESSFUL_MESSAGE_INVALID_TAG;
```

File: src/main/java/seedu/duke/command/ListNoteCommand.java [hunk 3]
```diff
@@ -137,9 +137,9 @@ public String execute() {
             }
 
             if (noteString.toString().isBlank()) {
-                return COMMAND_UNSUCCESSFUL_MESSAGE_EMPTY_NOTEBOOK;
+                return InterfaceManager.LS + COMMAND_UNSUCCESSFUL_MESSAGE_EMPTY_NOTEBOOK;
```
Comment (Chongjx) [inline src/main/java/seedu/duke/command/ListNoteCommand.java:140]:
```suggestion
                return Formatter.LS + COMMAND_UNSUCCESSFUL_MESSAGE_EMPTY_NOTEBOOK;
```

File: src/main/java/seedu/duke/command/ListNoteCommand.java [hunk 4]
```diff
@@ -137,9 +137,9 @@ public String execute() {
...
             }
-            return COMMAND_SUCCESSFUL_MESSAGE + noteString.toString();
+            return InterfaceManager.LS + COMMAND_SUCCESSFUL_MESSAGE + noteString.toString();
```
Comment (Chongjx) [inline src/main/java/seedu/duke/command/ListNoteCommand.java:142]:
```suggestion
            return Formatter.LS + COMMAND_SUCCESSFUL_MESSAGE + noteString.toString();
```

File: src/main/java/seedu/duke/Duke.java [hunk 5]
```diff
@@ -20,9 +20,34 @@
     private Timetable timetable;
     private TagManager tagManager;
 
+    // Character code adapted from http://patorjk.com/software/taag/#p=display&f=Ghost&t=NotUS
+    // Slight modifications made to make it easier on the eyes
+    private static final String NOTUS_LOGO = InterfaceManager.LS
```
Comment (Chongjx) [inline src/main/java/seedu/duke/Duke.java:25]:
```suggestion
    private static final String NOTUS_LOGO = Formatter.LS
```

File: src/main/java/seedu/duke/Duke.java [hunk 6]
```diff
@@ -20,9 +20,34 @@
...
+            + InterfaceManager.LS
+            + "|  .     |/ \\_) |  | |  |   |  |   |  | |( OO ) '..`''.) "
+            + InterfaceManager.LS
+            + "|  |\\    |    \\ |  | |  |   |  |   |  | | `-' /.-._)   \\ "
+            + InterfaceManager.LS
+            + "|  | \\   |     `'  '-'  '   |  |  ('  '-'(_.-' \\       / "
+            + InterfaceManager.LS
+            + "`--'  `--'       `-----'    `--'    `-----'     `-----'  "
+            + InterfaceManager.LS;
+
```
Comment (Chongjx) [inline src/main/java/seedu/duke/Duke.java:45]:
```suggestion
            + Formatter.LS
            + "     .-') _               .-') _                 .-')    "
            + Formatter.LS
            + "    ( OO ) )             (  OO) )               ( OO ).  "
            + Formatter.LS
            + ",--./ ,--,'  .-'),-----. /     '._ ,--. ,--.   (_)---\\_) "
            + Formatter.LS
            + "|   \\ |  |\\ ( OO'  .-.  '|'--...__)|  | |  |   /    _ |  "
            + Formatter.LS
            + "|    \\|  | )/   |  | |  |'--.  .--'|  | | .-') \\  :` `.  "
            + Formatter.LS
            + "|  .     |/ \\_) |  | |  |   |  |   |  | |( OO ) '..`''.) "
            + Formatter.LS
            + "|  |\\    |    \\ |  | |  |   |  |   |  | | `-' /.-._)   \\ "
            + Formatter.LS
            + "|  | \\   |     `'  '-'  '   |  |  ('  '-'(_.-' \\       / "
            + Formatter.LS
            + "`--'  `--'       `-----'    `--'    `-----'     `-----'  "
            + Formatter.LS;

```

File: src/main/java/seedu/duke/Duke.java [hunk 7]
```diff
@@ -20,9 +20,34 @@
...
     private static final String WELCOME_MSG_STRING = "Welcome to NotUS! "
+            + NOTUS_LOGO + InterfaceManager.LS
```
Comment (Chongjx) [inline src/main/java/seedu/duke/Duke.java:47]:
```suggestion
            + NOTUS_LOGO + Formatter.LS
```

File: src/main/java/seedu/duke/Duke.java [hunk 8]
```diff
@@ -20,9 +20,34 @@
...
             + "Type \"help\" if you need to see a list of commands and their usages.";
-    private static final String ENTER_COMMAND_MSG = "Enter command:";
+    private static final String ENTER_COMMAND_MSG = InterfaceManager.LS
```
Comment (Chongjx) [inline src/main/java/seedu/duke/Duke.java:49]:
```suggestion
    private static final String ENTER_COMMAND_MSG = Formatter.LS
```

**Review by Chongjx — CHANGES_REQUESTED** [review]
I moved LS over to Formatter!

File: src/main/java/seedu/duke/Duke.java [hunk 9]
```diff
@@ -20,9 +21,34 @@
...
+            + Formatter.LS
+            + "|  |\\    |    \\ |  | |  |   |  |   |  | | `-' /.-._)   \\ "
+            + Formatter.LS
+            + "|  | \\   |     `'  '-'  '   |  |  ('  '-'(_.-' \\       / "
+            + Formatter.LS
+            + "`--'  `--'       `-----'    `--'    `-----'     `-----'  "
+            + Formatter.LS;
+
     private static final String WELCOME_MSG_STRING = "Welcome to NotUS! "
+            + NOTUS_LOGO + Formatter.LS
```
Comment (Chongjx) [inline src/main/java/seedu/duke/Duke.java:48]: Can move this over to formatter class, and have a static function to get this.

**Review by Chongjx — APPROVED** [review]
Looks good. I will be updating the use Formatter in the next issue

**Comment by Chongjx** [comment]
Btw can we add the .log files into gitignore. Everyone can generate their .log file on their own local copy.

**Comment by R-Ramana** [comment]
> Btw can we add the .log files into gitignore. Everyone can generate their .log file on their own local copy.

Can. will make the changes in my next commit.

File: src/main/java/seedu/duke/Duke.java (hunk 9 above)
Comment (R-Ramana) [reply src/main/java/seedu/duke/Duke.java:48]: Will do so with my next PR!

**Pull request merged**